src/main/scala/com/microsoft/hyperspace/index/IndexLogManager.scala

```diff
@@ -71,8 +72,16 @@ class IndexLogManagerImpl(indexPath: Path, hadoopConfiguration: Configuration =
     if (!fs.exists(path)) {
       return None
     }
+
     val contents = FileUtils.readContents(fs, path)
-    Some(LogEntry.fromJson(contents))
+
+    try {
+      Some(LogEntry.fromJson(contents))
+    } catch {
+      case e: Exception =>
+        throw HyperspaceException(
+          s"Cannot parse json with file path: ${path}, reason: ${e.getMessage}")
```

Comment:
No specific rule, but for consistency:
```suggestion
          s"Cannot parse JSON in ${path}: ${e.getMessage}")
```

Reply:
Ok. Please let me know if the new title doesn't follow the consistency :)